src/superphot_plus/utils.py

```diff
@@ -36,12 +37,11 @@ def get_band_extinctions(ra, dec):
     coords = SkyCoord(ra, dec, frame="icrs", unit="deg")
     Av_sfd = 2.742 * sfd(coords)  # from https://dustmaps.readthedocs.io/en/latest/examples.html
 
-    # for gr, the was are:
-    band_wvs = 1.0 / (0.0001 * np.asarray([4741.64, 6173.23]))  # in inverse microns
+    band_wvs = 1.0 / (0.0001 * np.asarray(wvs)) # in inverse microns
 
     # Now figure out how much the magnitude is affected by this dust
-    ext_list = extinction.fm07(band_wvs, Av_sfd, unit="invum")  # in magnitudes
-
+    ext_list = extinction.fm07(band_wvs, Av_sfd, unit="invum") # in magnitudes
+    
```

Comment:
There are a couple small formatting items that can be cleaned up by running the `black .` from the command line.

Alternatively, you can manually delete the extra spaces on line 44.